reference-architecture/osp-cli/ch4.4_security_groups.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh
+
+neutron security-group-create bastion-sg
+neutron security-group-create master-sg
+#neutron security-group-create etcd-sg
+#neutron security-group-create router-sg
+neutron security-group-create infra-sg
+neutron security-group-create node-sg
+
+
+
+#
+# Bastion:
+#  Port/Proto  From      Reason
+#
+#  22/TCP      anywhere  Secure Shell
+#  53/TCP      anywhere  Nameservice
+#  53/UDP      anywhere  Nameservice
+neutron security-group-rule-create bastion-sg --protocol icmp
+neutron security-group-rule-create bastion-sg \
+    --protocol tcp --port-range-min 22 --port-range-max 22
+neutron security-group-rule-create bastion-sg \
+    --protocol tcp --port-range-min 53 --port-range-max 53
+neutron security-group-rule-create bastion-sg \
+    --protocol udp --port-range-min 53 --port-range-max 53
+
+
+#
+# Master:
+#   Port/Proto  From      Reason
+#
+#  22/TCP      bastion   Secure Shell
+#  8443/TCP    lb        WebUI/REST
+#  8443/TCP    master    WebUI/REST     ??
+#  8443/TCP    node      WebUI/REST     ??
+#  8053/TCP    node      Name Service   ??
+#  8053/UDP    node      Name Service   ??
+neutron security-group-rule-create master-sg --protocol icmp
+for PORT in 22 53 2379 4001 4789 8053 8443 10250 24224 ; do
```

Comment:
I'm not sure that 8053 needs to be exposed on the master (I could be wrong, though) @sdodson can you confirm?